sdk/storagecache/Azure.ResourceManager.StorageCache/samples/Generated/Samples/Sample_StorageCacheImportJobCollection.cs

```diff
@@ -100,7 +100,7 @@ public async Task Get_ImportJobsGet()
         public async Task GetAll_ImportJobsListByAmlFilesystem()
         {
             // Generated from example definition: specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2024-03-01/examples/importJobs_ListByAmlFilesystem.json
-            // this example is just showing the usage of "importJobs_ListByAmlFileSystem" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "importJobs_ListByAmlFilesystem" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `Filesystem`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2024-03-01/amlfilesystem.json#L767